recipe/meta.yaml

```diff
@@ -33,8 +33,7 @@ requirements:
     - python-dateutil
     - myproxyclient
```

Comment:
Also remove the version requirement for numpy since we fixed that incompatibility with this release.